.gitignore

```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /.pnp
+/package-lock.json
```

Comment:
also for some reasons the package lock is not being ignore.

Reply:
I added "package-lock.json" to the .gitignore